fog/overseer/server/tests/get_ingest_summaries.rs

```diff
@@ -0,0 +1,121 @@
+// Copyright (c) 2018-2022 The MobileCoin Foundation
+
+use mc_common::logger::{test_with_logger, Logger};
+use mc_fog_overseer_server::{server, server::OverseerState, service::OverseerService};
+use mc_fog_sql_recovery_db::test_utils::SqlRecoveryDbTestContext;
+use mc_ledger_db::{Ledger, LedgerDB};
+use mc_transaction_core::{Block, BlockContents};
+use mc_watcher::watcher_db::WatcherDB;
+use rand_core::SeedableRng;
+use rand_hc::Hc128Rng;
+use regex::Regex;
+use rocket::local::Client;
+use std::time::Duration;
+use tempdir::TempDir;
+
+mod utils;
+
+const PORT_NUMBER: u16 = 8086;
+
+// Ensures that the correct ingest summaries are produced when the Fog Ingest
+// cluster has one active node.
+#[test_with_logger]
+fn one_active_node_produces_ingest_summaries(logger: Logger) {
+    let db_test_context = SqlRecoveryDbTestContext::new(logger.clone());
+    let recovery_db = db_test_context.get_db_instance();
+    let mut rng: Hc128Rng = SeedableRng::from_seed([0u8; 32]);
+
+    // Set up the watcher db
+    let blockchain_path =
+        TempDir::new("blockchain").expect("Could not make tempdir for blockchain state");
+    let watcher_path = blockchain_path.path().join("watcher");
+    std::fs::create_dir(&watcher_path).expect("couldn't create dir");
+    WatcherDB::create(&watcher_path).unwrap();
```

Comment:
Optional/Equivalent: Replace `unwrap` with `expect` for this setup.

Reply:
This is used throughout the codebase, so will leave as is. We can revisit this in a small follow-up PR if we so desire.